contracts/VoteDelegation.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.13;
+import "./interfaces/IVotingEscrow.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract VoteDelegation is Ownable {
```

Comment:
I don't see this interacting with any of the other contracts, where is it in the diagram? (also maybe a vyper thing to look into)